tests/python/integration/pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+usefixtures = plugin dss_target
```

Comment:
what is this for ?

Reply:
This is required to run the integrations tests. It is copied from the plugin repo template.